### PR TITLE
[Toolchain] Auto refresh toolchain view when toolchain is installed

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit updates the toolchain view automatically when toolchain
is installed.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Comment by jyoungyun** [comment]
Dependencies on #704

**Comment by hyunsik-yoon** [comment]
Conflict... 😿

**Review by hyunsik-yoon — APPROVED** [review]

**Comment by jyoungyun** [comment]
ah. this is no merge PR. :)

**Comment by jyoungyun** [comment]
PTAL /cc @Samsung/one-vscode

File: src/View/InstallQuickInput.ts [hunk 1]
```diff
@@ -107,5 +107,15 @@ export async function showInstallQuickInput(context: vscode.ExtensionContext) {
   const state = await collectInputs();
   console.log(
       `Selected backend: ${state.backend.label}-${state.toolchainType}-${state.version.label}`);
-  state.toolchainEnv.install(state.toolchain);
+
+  return new Promise((resolve, reject) => {
```
Comment (YongseopKim) [inline src/View/InstallQuickInput.ts:111]: How about using your `JobCallback` landed?

```ts
  return new Promise((resolve, reject) => {
    const success: JobCallback = function () { resolve(state.toolchain); };
    const failed: JobCallback = function () { reject(); };
    state.toolchainEnv.install(state.toolchain, success, failed);
  });
```

File: src/View/InstallQuickInput.ts [hunk 2]
```diff
@@ -107,5 +107,15 @@ export async function showInstallQuickInput(context: vscode.ExtensionContext) {
...
+    state.toolchainEnv.install(
+        state.toolchain,
+        () => {
+          resolve(state.toolchain);
+        },
+        () => {
+          reject();
+        });
```
Comment (jyoungyun) [inline src/View/InstallQuickInput.ts:119]:
```suggestion
  return new Promise((resolve, reject) => {
    const success: JobCallback = function() {
      resolve(state.toolchain);
    };
    const failed: JobCallback = function() {
      reject();
    };
    state.toolchainEnv.install(state.toolchain, success, failed);
  });
```

File: src/View/InstallQuickInput.ts [hunk 3]
```diff
@@ -15,11 +15,13 @@
  */
 
 import * as vscode from 'vscode';
+
```
Comment (jyoungyun) [inline src/View/InstallQuickInput.ts:18]: The code formatter adds this new line.

**Comment by jyoungyun** [comment]
PTAL /cc @YongseopKim @hyunsik-yoon

**Review by hyunsik-yoon — APPROVED** [review]

**Review by YongseopKim — APPROVED** [review]
LGTM

**Pull request merged**